### PR TITLE
Determine js_bundle path from static.json, if present

### LINKED ISSUE #7: Determine js_bundle path from static.json, if present
The js_bundle path is [currently hardcoded](https://github.com/mars/create-react-app-inner-buildpack/blob/master/.profile.d/inject_react_app_env.sh#L10):
```sh
js_bundle=/app/build/static/js/main.*.js
```
However, the outer buildpack [may set a different root via `static.json`](https://github.com/mars/create-react-app-buildpack#web-server), which causes the js_bundle to be produced at a different path.

The js_bundle path should be derived from the content of `/app/static.json`'s `root` if present and retain the current behavior if not.
```sh
js_bundle=/app/${static.json.root}/build/static/js/main.*.js
```

Without this change, the following error results at runtime from sites deployed to Heroku, as the `REACT_APP_VARS_AS_JSON` token is not replaced.
```
Runtime env vars cannot be parsed. Content is `{{REACT_APP_VARS_AS_JSON_______…`
```

### PR DESCRIPTION
**Pull request by jared-dykstra**
Fixes #7

### CODE REVIEW

File: .profile.d/inject_react_app_env.sh [hunk 1]
```diff
@@ -5,9 +5,17 @@ set -e
...
 #set -x
 
-# Each bundle is generated with a unique hash name
-# to bust browser cache.
-js_bundle=/app/build/static/js/main.*.js
+static_json=/app/static.json
+if [ -f $static_json ]
+then
+  echo "Resolving static bundle from $static_json"
+  static_root=$(/app/.heroku/node/bin/node -pe 'JSON.parse(process.argv[1]).root || ""' "$(cat $static_json)")
```
Comment (mars) [inline .profile.d/inject_react_app_env.sh:12]: This parses to JSON with Node, which sets up a dependency on that buildpack. That is why the bundle injection script (below) is written in Ruby.

Please switch this to Ruby for JSON parsing.

**Review by mars — DISMISSED** [review]
See first my comments on #7 👏👏👏

Please switch JSON parsing from `node` to `ruby`.

Also contingent on my own progress with generalizing JS Runtime Env.

Thank you Jared.

File: .profile.d/inject_react_app_env.sh (hunk 1 above)
Comment (mars) [reply .profile.d/inject_react_app_env.sh:12]: Why Ruby? Ruby is included and available by default in the Heroku stack image. Node is not, requires a buildpack.

Comment (jared-dykstra) [reply .profile.d/inject_react_app_env.sh:12]: I have zero ruby experience, sorry.

**Comment by mars** [comment]
[Here's a new version of this PR](https://github.com/mars/create-react-app-inner-buildpack/pull/12) that uses Ruby for JSON parsing and also honors the new [`JS_RUNTIME_TARGET_BUNDLE` config](https://github.com/mars/create-react-app-buildpack/blob/master/README.md#user-content-custom-bundle-location).

It's not clear to me how you end up needing this special case, because simply setting `root` in **static.json** does not change the build location of **create-react-app**.

If this is something that is useful to folks, please leave feedback on #12

**Pull request closed without merging**